src/net.cpp

```diff
@@ -2853,7 +2855,16 @@ int64_t CConnman::PoissonNextSendInbound(int64_t now, int average_interval_secon
 
 int64_t PoissonNextSend(int64_t now, int average_interval_seconds)
 {
-    return now + (int64_t)(log1p(GetRand(1ULL << 48) * -0.0000000000000035527136788 /* -1/2^48 */) * average_interval_seconds * -1000000.0 + 0.5);
+    const int64_t average_interval_ms = average_interval_seconds * 1000000ULL;
```

Comment:
math isn't my strongest suit: Why are we multiplying seconds by 100k to get ms? 🤔

Reply:
> math isn't my strongest suit: Why are we multiplying seconds by 100k to get ms? thinking

Why `1000000ULL` is 100k? :)

Reply:
Going to update names.

Reply:
Thanks! Updated.

Reply:
too many zeroes clearly :eyes: , thanks for update of name